libc/watcom/asm/segments.asm

```diff
@@ -107,57 +58,4 @@ _End_YI label byte
         public  "C",_End_YI
 YIE     ends
 
-_BSS    segment word public 'BSS'
-        ;extrn   _edata                  : byte  ; end of DATA (start of BSS)
-        ;extrn   _end                    : byte  ; end of BSS (start of STACK)
-_BSS    ends
-
-;STACK_SIZE      equ     1000h
-;STACK   segment para stack 'STACK'
-        ;db      (STACK_SIZE) dup(?)
-;STACK   ends
-
-        assume  nothing
-        public  _cstart_
-
-        assume  cs:_TEXT
-
-_cstart_ proc near
-        dw      __begtext               ; make sure dead code elimination
-                                        ; doesn't kill BEGTEXT segment
-_cstart_ endp
```

Comment:
Even though `_cstart_` is not longer used (thank you), we need a way too ensure that dead code elimination doesn't remove the `__begtext` reference, so that (int)function pointers > 2. That is why I took advantage of the default OWC `_cstart_` symbol and left this in.

Reply:
It is clear I will do necessary change for these guards to works properly.
But be carefull it depends on memory model used for code.
You declared that libc is using large memory model that I will suppose only large memory model or do you expect libc for multiple memory models?

Reply:
For the time being, the OW memory model is set by reading libc/watcom.model. Small, Compact, Medium and Large models are supported and all work.

`_cstart_` can be removed here, but we need a way to ensure that the BEGTEXT segment is included with 2 bytes used so that `signal` function works properly and no function address == 0 or 1. The `__begtext` symbol and usage can be removed if you can think of a better way to force the linker to include the segment, even with dead code optimization is on. If this cannot be done, then we can possibly reference `__begtext` or a better name from crt0.c in stead of using `_cstart_` kluge.